src/Assignment5/UtilityApp/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace UtilityApp
+{
+    /// <summary>
+    /// Valida class has main method.
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main method.
+        /// </summary>
+        public static void Main()
+        {
+        }
+
+        /// <summary>
+        /// Message has the exit message.
+        /// </summary>
+        public static void Message()
```

Comment:
Is this method needed here or can it be moved to some other class?

Reply:
The method is moved to another class `MessageDisplayer`